themes/default/source/css/components/quire-citation.scss

```diff
@@ -63,20 +63,20 @@
     z-index: 1;
     bottom: 150%;
     left: 50%;
-    margin-left: -200px;
+    transform: translateX(-200px);
     -webkit-box-shadow: 0px 0px 6px 0px $black-semi-transparent;
     -moz-box-shadow: 0px 0px 6px 0px $black-semi-transparent;
     box-shadow: 0px 0px 6px 0px $black-semi-transparent;
     animation: .25s ease-out 0s 1 fadeIn;
     @media screen and (max-width: $desktop) {
       width: auto;
       right: 0;
-      margin-left: -100px;
+      transform: translateX(-100px);
     }
     @media screen and (min-width: $desktop) and (max-width: 978px) {
       width: 200px;
       right: 0;
-      margin-left: -100px;
+      transform: translateX(-100px);
```

Comment:
@hbalenda Having `width: auto` on line 72 doesn't work here, the citation box defaults to being very very skinny, at least on Firefox. 

My fix was to set that width to 200px, and then just delete lines 76–80. One break point is enough.

Reply:
@geealbers Yep, totally, I can make those changes today and will tag you. In the future, if the PR/task is closed, it's preferable to open a new JIRA ticket and assign to me so I can track the work.